extensions/amp-action-macro/amp-action-macro.md

```diff
@@ -1,5 +1,5 @@
 <!---
-Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Don't update copyright years for existing files. Only use current year for new files.